resources/rpm/project.spec.erb

```diff
@@ -233,7 +232,9 @@ fi
       chkconfig --del <%= service.name %> || :
     fi
   <%- elsif @platform.servicetype == "aix" -%>
-    /usr/bin/stopsrc -s <%= service.name -%> > /dev/null 2>&1 || :
+    if [ $1 = 0 ]; then
```

Comment:
`=` is strict string equality. But in shell, basically everything is a string. No strong feeling for "correctness" but maybe consistency?